go/noise/laplace_noise.go

```diff
@@ -187,6 +204,22 @@ func checkArgsLaplace(label string, l0Sensitivity int64, lInfSensitivity, epsilo
 	return checks.CheckNoDelta(label, delta)
 }
 
+func checkArgsConfidenceIntervalLaplace(label string, l0Sensitivity int64, lInfSensitivity, epsilon, delta, alpha float64) error {
+	if err := checks.CheckAlpha(label, alpha); err != nil {
+		return err
+	}
+	if err := checks.CheckL0Sensitivity(label, l0Sensitivity); err != nil {
+		return err
+	}
+	if err := checks.CheckLInfSensitivity(label, lInfSensitivity); err != nil {
+		return err
+	}
+	if err := checks.CheckEpsilonStrict(label, epsilon); err != nil {
+		return err
+	}
+	return checks.CheckNoDelta(label, delta)
+}
+
```

Comment:
I thought checks.go is for more general checks where it can be used in multiple files. However, this check is specifically for Laplace distribution and I thought it is better for it to be in this file. Also, this what was done for argument checking in the AddNoise(..) functions
https://github.com/google/differential-privacy/blob/a993d2ae8c5486634559f555647ce169e196bd7a/go/noise/laplace_noise.go#L177